src/clr-addons/location-bar/location-bar-node/location-bar-node.component.html

```diff
@@ -1,73 +1,67 @@
-<ng-container *ngIf="parentNode">
-  <ng-content *ngIf="parentNode.getChildren()?.length > 0"></ng-content>
-
-  <ng-container
-    *ngIf="
-      (selectableChilds.length > 0 && !parentNode.getSelectedChild()) || selectableChilds.length > 1;
-      else singleSelected
-    "
+@if (parentNode) { @if (parentNode.getChildren()?.length > 0) {
+<ng-content></ng-content>
+} @if ( (selectableChilds.length > 0 && !parentNode.getSelectedChild()) || selectableChilds.length > 1) {
+<clr-dropdown>
+  <button class="btn btn-link btn-sm" (blur)="focus = false" (focus)="focus = true" clrDropdownTrigger>
+    @if (parentNode.getSelectedChild(); as node) {
+    <span
+      class="text-truncate"
+      [class.last-level]="!parentNode.getSelectedChild().getSelectedChild()"
+      title="{{ node.label }}"
+    >
+      {{ node.label }}
+    </span>
+    <cds-icon shape="angle" direction="down" size="xs" class="caret-icon"></cds-icon>
+    } @else {
+    <cds-icon class="unselected" size="18" [attr.solid]="focus" shape="tree-view"></cds-icon>&nbsp;... }
+    <ng-template #unselected>
+      <cds-icon class="unselected" size="18" [attr.solid]="focus" shape="tree-view"></cds-icon>&nbsp;...
+    </ng-template>
+  </button>
+  <clr-dropdown-menu [clrPosition]="'bottom-left'" *clrIfOpen>
+    @for (item of selectableChilds; track item) {
+    <button
+      type="button"
+      clrDropdownItem
+      class="text-truncate"
+      [class.selected-child]="item.id === parentNode.getSelectedChild()?.id"
+      (click)="selectNode(item)"
+      title="{{ item.label }}"
+    >
+      {{ item.label }}
+    </button>
+    }
+  </clr-dropdown-menu>
+</clr-dropdown>
+} @else { @if (parentNode.getSelectedChild()) { @if (parentNode.getSelectedChild().selectable) {
```

Comment:
we should do something about these multi-level control blocks on the same line, does Prettier not format this in a nicer way?

Reply:
@thred did you run the control-flow migration on purpose? I think we can leave this until Angular 18

Reply:
Prettier will be fixed with 3.3.1, but currently only the version 3.2.5 is available. We'll wait for it and perform the migration at a later date ...